packages/esm-service-queues-app/src/active-visits/active-visits-table.resource.ts

```diff
@@ -105,8 +107,8 @@ export interface MappedVisitQueueEntry {
   queueComingFrom: string;
 }
 
-interface UseVisitQueueEntries {
-  visitQueueEntries: Array<MappedVisitQueueEntry> | null;
+interface UseVisitQueueEntries<T extends MappedVisitQueueEntry | VisitQueueEntry> {
```

Comment:
I haven't dove deeplying the queue module yet, so I'm just guessing about this, but if this is anything like the Appointment module, do we want to try to standardized on a single queue entry interface (probably VisitQueueEntry and retire the MappedVisitQueue entry?).  Is a lot of this code just stuff to support translating between a mapped and "normal" visit queue entry? (The answer may be yes, but we don't want to tackle it in this commit).

Reply:
Update: I see the later comments about getting rid of this, +1